app/controllers/bookings_controller.rb

```diff
@@ -9,7 +9,7 @@ def new
   end
 
   def create
-    utc_date = date_params.to_s.to_time.utc
+    utc_date = booking_params.to_s.to_time.utc
     booked_date = DateTime.parse(utc_date.to_s)
     @booking = Booking.new(user_id: current_user.id, kondo_id: params[:kondo_id], booked_date: booked_date)
```

Comment:
I just merged @spioupiou's pr [45](https://github.com/spioupiou/is-kondo/pull/45/files) which cause the conflict on this line. We need to keep the format Cédrine used otherwise there will be an error.
```erb
@booking = Booking.new(user_id: current_user.id, kondo_id: params[:kondo_id], booked_date: params[:booking][:booked_date])
```

Reply:
I would just remove line 13-14-15 and keep line 17. Then merge Hiro's branch and ruin his magnificent code with ugly if-else statements to cater for unlogged users. Then we have a good basis to work on tomorrow :) 

![Screen Shot 2022-01-28 at 18 16 24](https://user-images.githubusercontent.com/68413600/151520308-3ad6cfd3-4e9e-4ebf-8439-5a89161c35c6.png)